main.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/242e63f88a.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@100;300;400;500;700;900&family=Poppins:wght@100;400;500;700;900&family=Roboto:wght@100;400;500;700;900&display=swap" rel="stylesheet">    
+    <title>Portfolio</title>
+</head>
+<body>
+    <section id="main-section">
+        <div id="bar-container">
+            <ul id="status-bar">
+                <li class="time">9:41</li>
+                <li class="fas-container">
+                    <i class="fas fa-signal"></i>
+                    <i class="fas fa-wifi"></i>
+                    <i class="fas fa-battery-full"></i>
+                </li>
+            </ul>
+            <header id="tool-bar">
+                <a class="logo" href="#">John Doe</a>
+                <i class="fas fa-bars"></i>
+            </header>
+        </div>
+        <div id="intro">
+            <h1 class="header1">Hey There.<br>I’m Jordan</h1>
+            <h3 class="header2">I’am a Software Developer</h3>
+            <p class="description">
+                I can help you build a product , feature or website 
+                Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <ul id="social-media-links">
+                <li class="link">
+                    <a href="https://github.com/Jord4N-AC"><img src="images/Icons/github_icon.svg" alt="github icon"></a>
+                </li>
+                <li class="link">
+                    <a href="https://www.linkedin.com/in/camilo-armijos-2b9648197/"><img src="images/Icons/linkedin_icon.svg" alt="linkedin icon"></a>
+                </li>
+                <li class="link">
+                    <a href="https://www.angellist.com/"><img src="images/Icons/angelist_icon.svg" alt="angelist icon"></a>
+                </li>
+                <li class="link">
+                    <a href="https://twitter.com/jordan12AC"><img src="images/Icons/twitter_icon.svg" alt="twitter icon"></a>
+                </li>
+                <li class="link">
+                    <a href="https://medium.com/"><img src="images/Icons/medium_icon.svg" alt="medium_icon"></a>
+                </li>
+            </ul>
+        </div>
+    </section>    
```

Comment:
Kindly separate between the two sections, make the first one with the `header` tag, and the second one with the `section` tag.
Like 
```
<header>
  <nav>.....</nav>
</header>
<section>
 ......
</section>
```